src/pages/nextStepPropTypes.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types';
+
+const messagePropType = PropTypes.shape({text: PropTypes.string, type: PropTypes.string, action: PropTypes.string});
```

Comment:
Can we start each key from a new line.
```suggestion
const messagePropType = PropTypes.shape({
  text: PropTypes.string, 
  type: PropTypes.string, 
  action: PropTypes.string
});
```